libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -230,6 +230,8 @@ class AC_WPNav
     AP_Float    _terrain_margin;        // terrain following altitude margin. vehicle will stop if distance from target altitude is larger than this margin
 
     float _last_wp_speed_cms;  // last recorded WPNAV_SPEED, used for changing speed in-flight
+    float _last_wp_speed_up_cms;  // last recorded WPNAV_SPEED, used for changing speed in-flight
```

Comment:
I guess we should update the comments to reflect the corresponding parameter names ..

Reply:
Done